web/routes/web.php

```diff
@@ -123,6 +123,32 @@
 
 // -- WIKI
 
+// (names, not route paths)
+// TODO: account
+// TODO: account.profile
+// TODO: account.settings
+// TODO: account.messages
+// TODO: account.notifications
+// TODO: user.profile
+// TODO: docs
+// TODO: terms
+// TODO: security
+// TODO: privacy
+// TODO: report-flag
+
+// dummy routes so that these resolve on the frontend
+Route::view('--/account', 'next.test.page-test')->name('account');
```

Comment:
What does `--` mean?

Reply:
`--` is an invalid Wikidot route, right? I just picked something that wouldn't conflict.

I actually kind of like it because then I can put anything after it, like `login` or `register`. `user--services` is unwieldy, and we're gonna have to come up with equivalents for several categories (like accounts).

Reply:
Makes sense. I was curious if it was a Laravel/Ziggy thing.